x-pack/elastic-agent/docs/stop-elastic-agent.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[stop-elastic-agent]]
+[role="xpack"]
+= Stop {agent}
+
+To stop {agent} and its related executables, stop the {agent} process.
+
+On Windows, if you installed the Agent as a service, stop the service. If
+necessary, use Task Manager on Windows to stop {agent}. This will kill the
+{agent} process and any sub-processes it created (such as {beats}).
+
+On Linux or macOS, run the following command to get the ID of the
+`elastic-agent` process:
```

Comment:
Elastic Agent also has .deb and .rpm installers on systemd system the service needs to be stopped.

`systemctl stop elastic-agent`